_posts/2017-02-09-creating-good-content.md

```diff
@@ -0,0 +1,66 @@
+---
+title: "Creating good content"
+category: blog
+tags: ['digital transformation']
+author: Libby Varcoe and Darren Menachemson
+author-excerpt: "Libby Varcoe is Content Community Lead at the Digital Transformation Agency
+Darren Menachemson is a Partner at ThinkPlace"
+thumbnail: /images/blog-thumbnails/Content-Design-Meet-Up-Thumb.jpg
```

Comment:
There's a typo, the file name of the thumbnail is Content-Design-Meet-Up-Thumb-1.jpg